runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -782,9 +787,18 @@ public void setResourceClient(XWalkResourceClientInternal client) {
      */
     @Override
     @XWalkAPI
-    public void setBackgroundColor(int color) {
+    public void setBackgroundColor(final int color) {
```

Comment:
Why add `final` here? I do not see android.view.View.setBackgroundColor has this change.